nislmigrate/migrators/file_migrator.py

```diff
@@ -38,6 +41,15 @@
 
 """
 
+_CLEAR_MIGRATION_DIRECTORY_BEFORE_CAPTURING_ERROR = """
+
+There is existing data in the migration directory that capturing would overwrite.
```

Comment:
I may be biased towards my testing process, but having to delete the data every time we migrate feels awkward. Can we not just overwrite?

If we want to enforce this, should we check for the whole root director being empty at the start of the capture operation, rather than enforcing this requirement in this specific plugin.

Reply:
I went ahead and removed the requirement that this file doesn't exist before capture.